src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagEntry.cs

```diff
@@ -0,0 +1,44 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Features.CustomTag
+{
+    /// <summary>
+    /// Represent each custom tag entry has retrieved from the store.
+    /// </summary>
+    public class CustomTagEntry
+    {
+        /// <summary>
+        /// Key of this custom tag entry.
+        /// </summary>
+        public long Key { get; set; }
+
+        /// <summary>
+        /// Path of this tag. Normally it's composed of groupid and elementid.
+        /// E.g: 00100020 is path of patient id.
+        /// </summary>
+        public string Path { get; set; }
```

Comment:
Do we need to do any sort of validation on the `Path` or `VR` properties anywhere? I assume that that's there's some format we expect these in.

Reply:
yes, CustomTagEntry is used when adding custom tag, the Path and VR will be validated over there.